Add user facing daily limits pages for emails, text messages, and letters

CrystalPea opened the pull request:
These pages are linked to from service's Settings pages.

## This is so:
- users see how much remaining limit they have
- users know to request extra limit and what info to provide

## To review:
- look at the code
- run the code locally
- content review [content designer]


## Future iterations:
- it would be great to add daily limit change request forms to the pages to get standarised data with the requests and further reduce back and forth on support

## Screenshots:

### Emails:
#### Link from settings:
![Screenshot 2025-07-03 at 16 59 22](https://github.com/user-attachments/assets/704b2545-ce0e-4177-99ca-34f6a7ca3340)
#### Limit page:
![Screenshot 2025-07-03 at 16 58 10](https://github.com/user-attachments/assets/f518d4a8-df6f-488c-8435-68377dc1ab0d)

### Text messages:
#### Link from settings:
![Screenshot 2025-07-03 at 16 58 18](https://github.com/user-attachments/assets/0eeafd79-c515-486e-8f1c-4e5d7fb76ed2)
#### Limit page:
![Screenshot 2025-07-03 at 16 58 47](https://github.com/user-attachments/assets/ef11a5f6-4d45-42af-9559-2d5f09e776e4)

### Letters
#### Link from settings:
![Screenshot 2025-07-03 at 16 59 01](https://github.com/user-attachments/assets/190e5257-c75a-464f-9317-568c3b402e6c)
#### Limit page:
![Screenshot 2025-07-03 at 16 59 13](https://github.com/user-attachments/assets/749fcc05-bfd5-4d20-9799-31ff3209f351)

karlchillmaid commented on app/templates/views/service-settings.html:
```diff
@@ -100,6 +100,13 @@
+      {% set email_limit_html %}
+        {{ "{} per day".format(current_service.get_message_limit("email")|format_thousands) }}
+        <div class="govuk-hint">
+          {{ current_service.sent_today("email")|string + " sent today" }}
```
Would it be more useful to see what you have left today, rather than what you‘ve sent?

For example:

![Screenshot 2025-07-03 at 23 47 49](https://github.com/user-attachments/assets/ece35f13-452b-46a7-b0b5-14826abd2e2d)

quis replied:
My reckon is that because the value you might want to change will be some multiple/addition/subtraction of the amount you’ve sent it’s better that the numbers are comparable (rather than doing the maths for you).

In other words these are all counting the same thing:
- old limit: 250,000
- sent today: 200,000
- new limit: 500,000 

Whereas these are counting different things:
- old limit: 250,000
- remaining: 50,000
- new limit: 500,000 

Once the user clicks through to the next page we will display sent and remaining, just to help confirm the limit works as they expect.

karlchillmaid commented on app/templates/views/service-settings/daily-message-limit.html:
```diff
@@ -0,0 +1,49 @@
+    <div class="govuk-grid-column-five-sixths">
+        {{ page_header(page_title) }}
+        <p class="govuk-body">
+            Number of {{ 3|message_count_noun(daily_limit_type) }} the service is allowed to send each day.
```
```suggestion
            There’s a limit to the number of {{ 3|message_count_noun(daily_limit_type) }} you can send each day.
```

karlchillmaid commented on app/templates/views/service-settings/daily-message-limit.html:
```diff
@@ -0,0 +1,49 @@
+
+        {{ ajax_block(partials, updates_url, 'remaining_limit') }}
+
+        <h2 class="heading-medium">Request an increase to your sending limit</h2>
```
```suggestion
        <h2 class="heading-medium">Change the daily {{ 1|message_count_noun(daily_limit_type) }} limit</h2>
```

karlchillmaid commented on app/templates/views/service-settings/daily-message-limit.html:
```diff
@@ -0,0 +1,49 @@
+
+        <h2 class="heading-medium">Request an increase to your sending limit</h2>
+
+        <p class="govuk-body">
+            If you need to discuss an increase to your daily {{ 1|message_count_noun(daily_limit_type) }} limit, <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.support') }}">contact us</a>.
+        </p>
```
```suggestion
        <p class="govuk-body">
            <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.support') }}">Contact us</a> if you need to increase to your daily {{ 1|message_count_noun(daily_limit_type) }} limit.
        </p>
```

CrystalPea commented on app/templates/views/service-settings.html:
```diff
@@ -202,7 +209,27 @@
+          {
+            "key": {
+              "classes": "notify-summary-list__key notify-summary-list__key--35-100",
+              "text": "Email limit",
```
Also it already says "250,000 per day" just next to it, so saying "Daily limit" in the row title, too, could be a bit repetitive?

CrystalPea replied:
Ok, do we want to change it, even though text messages have two different daily limits? 

If we do, what do we do with `Text message limit` and `International text message limit` rows?

karlchillmaid replied:
After discussion we have decided to leave this as it is for now.

We don’t have evidence that users will find any (slight) inconsistency between the Settings page and the H1 of the page where you change the daily limit confusing.

We might want to do a bigger review of the Settings page and child page H1 patterns at some point, but now is not the time!

CrystalPea commented:
I made following changes after reviews from @karlchillmaid , Ryan and @quis :

- the links to new daily limits pages are only visible to live services
- changed content for "Service is live" bit at the bottom of Settings page
- Separate content for requesting a new daily letter limit
- updated content on the Daily limit page following reviews by Karl and Ryan

## Updated daily limits pages:
![Screenshot 2025-07-04 at 17 06 47](https://github.com/user-attachments/assets/c14d48ac-1c55-4a39-ad39-e75bc88c3def)
![Screenshot 2025-07-04 at 17 06 58](https://github.com/user-attachments/assets/54b83bcd-bb9a-442d-beb6-30f2e3026d42)
![Screenshot 2025-07-04 at 17 07 09](https://github.com/user-attachments/assets/20183cf6-b2df-4bf1-b321-f1e8684a6392)

## Changed "Service is live" bit:
![Screenshot 2025-07-04 at 16 02 47](https://github.com/user-attachments/assets/c8f3576d-aac2-439f-af86-5e34c7ce55ec)

karlchillmaid commented on app/templates/views/service-settings/daily-message-limit.html:
```diff
@@ -0,0 +1,56 @@
+            {% if daily_limit_type == "letter" %}
+                <li>the start and end dates of your letter campaign</li>
+                <li>the total volume of letters your service will send during this time</li>
+                <li>the number of sheets per template (2 pages in your template is equivalent to 1 double-sided sheet)</li>
```
```suggestion
                <li>the number of pages per letter</li>
```

CrystalPea commented:
All requested changes resolved.

PR ready for re-review.

Screenshots for limit pages at the moment:

![Screenshot 2025-07-08 at 16 00 53](https://github.com/user-attachments/assets/08580c9d-82c1-422f-bcff-7a1d750efa58)
![Screenshot 2025-07-08 at 16 01 04](https://github.com/user-attachments/assets/927f883e-2aa2-482d-bf29-78795e39ce24)
![Screenshot 2025-07-08 at 16 00 38](https://github.com/user-attachments/assets/8fd69a53-6717-40d7-be8f-9321326efaf0)
